ui/app/components/app-footer.hbs

```diff
@@ -0,0 +1,26 @@
+<Hds::AppFooter as |AF|>
+  <AF.ExtraBefore>
+    {{#if this.isDevelopment}}
```

Comment:
Moved the local dev badge into the footer, otherwise this is just copied from `vault.hbs`